soc/nuvoton/npcx/npck3/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Nuvoton NPCK3 EC series
+
+# Copyright (c) 2025 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NPCK3
```

Comment:
the if must go below the symbol, this cannot work as it is. And actually a board should be added to this PR so check it works (if one was added, CI would fail)